aws/scalardl/example.scalardl_container.env

```diff
@@ -0,0 +1,18 @@
+# Environment variables for both scalar-ledger image and scalar-auditor image
+# https://github.com/scalar-labs/scalar/blob/master/ledger/conf/ledger.properties.tmpl
+SCALAR_DB_STORAGE=cassandra
+SCALAR_DB_CONTACT_POINTS=cassandra-lb.internal.scalar-labs.com
+SCALAR_DB_CONTACT_PORT=9042
+SCALAR_DB_USERNAME=cassandra
+SCALAR_DB_PASSWORD=cassandra
+
+# Environment variables for scalar-auditor image
+# https://github.com/scalar-labs/scalar/blob/master/auditor/conf/auditor.properties.tmpl
+SCALAR_DL_AUDITOR_LEDGER_HOST=scalar-ledger.example.com
```

Comment:
Is it OK to use `example.com` for the host domain?

Reply:
The host of the ledger should be provided by the user from an environment deployed elsewhere. `example.com` is just a placeholder.

Reply:
Where are the dns for auditor ledger host created? (ｍanually or terraform ?)
Is it possible to specify the `internal_domain` by default?

Reply:
The ledger server specified here should be deployed separately with another copy of the `scalar-terraform-examples` scripts, or manually. The user must specify a concrete `SCALAR_DL_AUDITOR_LEDGER_HOST` from the environment. We should not assume that the same `internal_domain` is used in both the auditor and the ledger.